fsi/component/kinds.go

```diff
@@ -5,12 +5,13 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
-	"gopkg.in/yaml.v2"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 
+	"gopkg.in/yaml.v2"
+
```

Comment:
we don't normally use spaces between import domains. IMHO should be:

```
import (
  std_lib_stuff

  everything_else
)
```

code review comments don't have much opinion on the grouping beyond this.
https://github.com/golang/go/wiki/CodeReviewComments#imports

One thing I've seen emerge in other projects is to group imports owned by the org